asdf_astropy/converters/nddata/uncertainty.py

```diff
@@ -0,0 +1,45 @@
+class _UncertaintyBaseConverter:
```

Comment:
I think this could all be done by one converter if you took the suggestion later to handle all this with one tag.